examples/source_file_geojson_raster.html

```diff
@@ -52,39 +52,70 @@
             itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
 
             var promises = [];
-
-            // Convert by iTowns
-            promises.push(itowns.Fetcher.json('https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson')
-                .then(function _(geojson) {
-                    return itowns.GeoJsonParser.parse(geojson, {
+            var optionsGeoJsonParser = {
                         buildExtent: true,
                         crsIn: 'EPSG:4326',
                         crsOut: view.tileLayer.extent.crs,
                         mergeFeatures: true,
                         withNormal: false,
                         withAltitude: false,
-                    });
+            };
+
+            // Convert by iTowns
+            promises.push(itowns.Fetcher.json('https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson')
+                .then(function _(geojson) {
+                    return itowns.GeoJsonParser.parse(geojson, optionsGeoJsonParser);
                 }).then(function _(parsedData) {
                     var ariegeSource = new itowns.FileSource({
                         parsedData,
                     });
 
-                    var ariegeLayer = new itowns.ColorLayer('ariege', {
-                        name: 'ariege',
-                        transparent: true,
-                        style: {
-                            fill: {
-                              color: 'orange',
-                              opacity: 0.5,
-                            },
-                            stroke: {
-                                color:'white',
+                    return itowns.Fetcher.texture('images/cross.png').then(texture => {
```

Comment:
Where is `texture` used here ?

Reply:
I use `texture fetcher` to load image, by using `texture.image`.

Reply:
Yes, but you don't use it for this layer, so you can remove it.

Reply:
ok